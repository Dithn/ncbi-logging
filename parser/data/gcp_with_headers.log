"time_micros","c_ip","c_ip_type","c_ip_region","cs_method","cs_uri","sc_status","cs_bytes","sc_bytes","time_taken_micros","cs_host","cs_referer","cs_user_agent","s_request_id","cs_operation","cs_bucket","cs_object"
"1554306916471623","35.245.77.223","1","","GET","/storage/v1/b/sra-pub-run-1/o/SRR002994%2FSRR002994.2?fields=name&alt=json&userProject=nih-sra-datastore&projection=noAcl","404","0","0","42000","www.googleapis.com","","apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","19919634438459959682894277668675","storage.objects.get","sra-pub-run-1","SRR002994/SRR002994.2"
"1588633215861145","35.245.231.141","1","","GET","/storage/v1/b/sra-pub-src-9/o?prefix=SRR10123568%2FMRP2_RH_15.fq.gz.1&projection=noAcl&fields=prefixes%2CnextPageToken%2Citems%2Fname&alt=json&userProject=nih-sra-datastore&delimiter=%2F&maxResults=1000","200","0","3","29000","www.googleapis.com","","apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","AAANsUkmFIqxJGcgvTOINNI2e3TxQnhBPtfNjaupBtM3LhUBDmRUFv_4fnJMBEH-7aKOrWAo_UXYmB2IUKsf7rJpJnI","","sra-pub-src-9",""
