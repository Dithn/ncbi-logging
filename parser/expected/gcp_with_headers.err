converting GCP format
total : 3
accepted : 2
rejected : 0
headers : 1
unrecognized : 0
