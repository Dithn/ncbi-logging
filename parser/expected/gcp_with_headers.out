{"fields":["time_micros","c_ip","c_ip_type","c_ip_region","cs_method","cs_uri","sc_status","cs_bytes","sc_bytes","time_taken_micros","cs_host","cs_referer","cs_user_agent","s_request_id","cs_operation","cs_bucket","cs_object"],"header":true}
{"accepted":true,"agent":"apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","bucket":"sra-pub-run-1","host":"www.googleapis.com","ip":"35.245.77.223","ip_region":"","ip_type":1,"method":"GET","object":"SRR002994/SRR002994.2","operation":"storage.objects.get","referer":"","request_bytes":0,"request_id":"19919634438459959682894277668675","result_bytes":0,"source":"GS","status":404,"time":1554306916471623,"time_taken":42000,"uri":"/storage/v1/b/sra-pub-run-1/o/SRR002994%2FSRR002994.2?fields=name&alt=json&userProject=nih-sra-datastore&projection=noAcl"}
{"accepted":true,"agent":"apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","bucket":"sra-pub-src-9","host":"www.googleapis.com","ip":"35.245.231.141","ip_region":"","ip_type":1,"method":"GET","object":"","operation":"","referer":"","request_bytes":0,"request_id":"AAANsUkmFIqxJGcgvTOINNI2e3TxQnhBPtfNjaupBtM3LhUBDmRUFv_4fnJMBEH-7aKOrWAo_UXYmB2IUKsf7rJpJnI","result_bytes":3,"source":"GS","status":200,"time":1588633215861145,"time_taken":29000,"uri":"/storage/v1/b/sra-pub-src-9/o?prefix=SRR10123568%2FMRP2_RH_15.fq.gz.1&projection=noAcl&fields=prefixes%2CnextPageToken%2Citems%2Fname&alt=json&userProject=nih-sra-datastore&delimiter=%2F&maxResults=1000"}
